defaults/main.yml

```diff
@@ -105,9 +105,8 @@ openio_oioswift_filter_cache:
     else \
     openio_oioswift_bind_address ~ ':6019' }}"
   memcache_max_connections: 10
-  # 0 means don't expire automatically
   oio_cache: true
-  oio_cache_ttl: 0
```

Comment:
https://github.com/open-io/customer-skeleton/blob/20.04/deployment/sds/collections/ansible_collections/openio/community/roles/oioswift/defaults/main.yml#L106

Can you PR this line ?